doc/introduction/introducing_zephyr.rst

```diff
@@ -8,7 +8,7 @@ resource-constrained systems: from simple embedded environmental sensors and LED
 wearables to sophisticated smart watches and IoT wireless gateways.
 
 The Zephyr kernel supports multiple architectures, including ARM Cortex-M, Intel
-x86, ARC, NIOS II, Tensilica Xtensa and RISC V. The full list of supported
+x86, ARC, NIOS II, Tensilica Xtensa and RISC-V. The full list of supported
```

Comment:
add the comma before and:

     Tensilica Xtensa, and RISC-V

Reply:
Done.